File: src/main/java/im/tox/tox4j/av/callbacks/ToxAvEventListener.java
package im.tox.tox4j.av.callbacks;

public interface ToxAvEventListener extends
    CallCallback,
    CallControlCallback,
    CallStateCallback,
    AudioBitRateStatusCallback,
    VideoBitRateStatusCallback,
    AudioReceiveFrameCallback,
    VideoReceiveFrameCallback {
  ToxAvEventListener IGNORE = new ToxAvEventAdapter();
}

File: projects/tox4j/src/main/java/im/tox/tox4j/av/callbacks/ToxAvEventListener.scala
package im.tox.tox4j.av.callbacks

trait ToxAvEventListener[ToxCoreState]
  extends CallCallback[ToxCoreState]
  with CallStateCallback[ToxCoreState]
  with BitRateStatusCallback[ToxCoreState]
  with AudioReceiveFrameCallback[ToxCoreState]
  with VideoReceiveFrameCallback[ToxCoreState]
